packages/realm/bindgen/package.json

```diff
@@ -3,6 +3,6 @@
   "private": true,
   "type": "module",
   "workspaces": [
-    "packages/realm/bindgen/vendor/realm-core/bindgen/"
+    "packages/realm/bindgen/vendor/realm-core/"
```

Comment:
Interesting. Any particular reason this needed to be moved to the root?

Reply:
Without that `npm run foo` and `npx bar` would only work when run from a subdirectory of `bindgen`. I thought it would be best if they worked when run from anywhere in the realm-core repo. https://github.com/realm/realm-core/commit/3bad9efef965406781de97daf4354b4181404627 _should_ solve this for commands run by cmake in the `$repo_root/build` directory, but it seemed better to just make those commands work from anywhere in the repo.

In theory, I could have made a root package.json, with bindgen as a subworkspace, but that seems silly for realm-core which isn't really a JS project.

Reply:
`npx bar` in particular would have the odd behavior of "working", but using the latest version of `bar` and its deps, rather than considering the versions from `package.json`

Reply:
Yeah. When not invoked through an NPM script in the package.json, `npx` use a "globally" installed package.